NewHorizons/Builder/Props/WarpPadBuilder.cs

```diff
@@ -122,13 +123,13 @@ public static void Make(GameObject planetGO, Sector sector, NomaiWarpReceiverInf
 
             if (info.computer != null)
             {
-                CreateComputer(planetGO, sector, info.computer, receiver);
+                CreateComputer(planetGO, sector, mod, info.computer, receiver);
             }
         }
 
-        public static void Make(GameObject planetGO, Sector sector, NomaiWarpTransmitterInfo info)
```

Comment:
is it worth commenting that mod is only used for Items? or is that obvious enough?

Reply:
Currently it's only used for item sounds, but there's always the possibility that it'll be used to look up other kinds of resources (textures, XMLs, whatever), so I wouldn't want to put in comments that could become incorrect in the future.